### PR TITLE
feat: Decreased feecap percentage

### PR DESCRIPTION
**Pull request by daviroo**
@jireva recently did some data analysis and worked out that the feecap percentage could be dropped to 0.5% of payment amount without effecting 99% of payments.  This PR drops the feecap to the appropriate level.

### CODE REVIEW

**Review by nicolasburtey — APPROVED** [review]

**Review by nicolasburtey — DISMISSED** [review]
actually not approving this as tests are failing now

**Comment by daviroo** [comment]
> actually not approving this as tests are failing now

It was actually failing because in the unit tests the new feecap was dropping the fee below the `FEEMIN` value currently set to 10 sats.  Should we also adjust the min fee value since we are decreasing the feecap?

**Comment by dolcalmi** [comment]
@vindard do we have 2 places to configure this? https://github.com/GaloyMoney/galoy/blob/main/src/domain/payments/ln-fees.ts#L9 would you mind to check?

File: test/unit/domain/shared/amount-calculator.spec.ts [hunk 1]
```diff
@@ -150,4 +150,15 @@ describe("AmountCalculator", () => {
       })
     }
   })
+
+  describe("multiplies", () => {
+    it("correctly applies basis points to an amount", () => {
```
Comment (dolcalmi) [inline test/unit/domain/shared/amount-calculator.spec.ts:155]: please add a test where the amount should be rounded and other which return min value (with a small amount)

**Review by dolcalmi — APPROVED** [review]

File: src/domain/shared/calculator.ts [hunk 2]
```diff
@@ -47,11 +47,18 @@ export const AmountCalculator = (): AmountCalculator => {
...
+  ) =>
+    divRound({ amount: amount.amount * basisPoints, currency: amount.currency }, 10_000n)
+
   return {
     sub,
     add,
     divRound,
     divFloor,
     divCeil,
+    applyBasisPoints,
```
Comment (nicolasburtey) [inline src/domain/shared/calculator.ts:62]:
```suggestion
    mulBasisPoints,
```

**Review by nicolasburtey — COMMENTED** [review]
one comment on naming

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**